packages/runtime/container-runtime/src/orderedClientElection.ts

```diff
@@ -262,12 +308,16 @@ export class OrderedClientElection
     ) {
         super();
         let initialClient: ILinkedClient | undefined;
+        let initialParent: ILinkedClient | undefined;
         for (const client of orderedClientCollection.getAllClients()) {
             this.addClient(client, 0);
             if (typeof initialState !== "number") {
```

Comment:
What does initialState = number imply? The comment above says "current sequence number at time of load if new" but I am not sure what it means.
If this client was just created / loaded how can it be the elected client or parent? Isn't that only possible if this is the very first "write" client in the document?

Reply:
Oh I get it. It's comparing the intialState's client and parent Id against all clients in the list and not against "this" client. My bad.

Reply:
"initialState === "number"" means that we didn't read an initially elected client from the snapshot, so the "initialState" just represents a sequence number. It may be true that today this only happens when there was no snapshot to load from. But note that the initial elected client is not the new one. It's whatever client was elected at the time of the snapshot. The point here, I think, is that all clients should have their OrderedClientElection's in the same state. There may be a better way to accomplish that. The way we're using today doesn't seem foolproof.